aws/createResources.sh

```diff
@@ -207,7 +208,13 @@ export const environment = {
     albumName: "usercontent",
     bucketRegion: '$REGION',
 
-    ddbTableName: '$TABLE_NAME'
+    ddbTableName: '$TABLE_NAME',
+
+    cognito_idp_endpoint: '$( if [ "$aws_cmd" == "awslocal" ]; then echo 'http://localhost:4590'; fi )',
```

Comment:
Defaulting to empty strings for these endpoints. (Handled accordingly in the code)